doc/ref_model/chapters/chapter01.md

```diff
@@ -57,10 +57,10 @@ This section defines the main terms used in this document; these deinitions are
 - **Network Function (NF)**:  functional block or application within a network infrastructure that has well-defined external interfaces and well-defined functional behaviour.
   - Within **NFV**, A **Network Function** is implemented in a form of **Virtualised NF** or a **Containerised NF**.
 - **Network Service (NS)**: composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
-- **Virtual Network Function (VNF)**: a software implementation of a **Network Function**, capable of running on the **NFVi**.
-  - **VNF**s are built from one or more VNF Components (**VNFC**) and, in most cases,  the VNFC is hosted on a single VM or Container.
-- **Cloud-native (containerised) Network Function (CNF)**: **VNF** with a full adherence to cloud native principles, or a **VNF** that is transitioning to cloud native. 
-  >_*Note:*_ It is a containerised **VNF** that is microservices-oriented, to increase agility and maintainability, and that can be dynamically orchestrated and managed to optimize resource utilization; the containers can be Linux, Docker or other similar container technology.
+- **Virtualised Network Function (VNF)**: a software implementation of a **Network Function** delivered as set of VMs capable of running on a **NFVI**.
```

Comment:
We should not create a definition that is different from the ETSI ISG NFV definition of a VNF, which is available in ETSI GS NFV 003. Not only is the ETSI definition not restricted to VM based VNFs, the VNF is not delivered as a set of VMs, but as a VNF package, containing e.g. a VNFD and images.

Reply:
The ETSI NFV definition of VNF ("Virtualised Network Function (VNF): implementation of an NF that can be deployed on a Network Function Virtualisation Infrastructure (NFVI)") *is* implicitly restricted to VM-based NFs, because one cannot run OS containers on a machine (compute resource) without the OS, a container runtime and (arguably) container orchestration system on top of the compute resource provided by the NFVI.

It seems the intention of the original author(s) was to differentiate between NFs deployed in VMs and those deployed in containers. Do you have another proposal how to extend ETSI NFV definition in a way that addresses this in a more compatible manner?

Reply:
The definition of the VNF is not focussing on the type of execution environment. It is should be seen from a MANO point of view. A VNF is explicitly meant to be agnostic of the used virtualization technology. E.g. it does not assume that there is no operating system below. Current ETSI NFV container study IFA029 which is currently in final review also does not change the definition of the VNF, but studies how to allow containerized workloads managed as VNFs.
If we need a term for a NF implemented in VMs and not in containers, we should use a new acronym, e.g. VMNF or V-VNF.

For the definition of CNF I recommend, that also MANO aspects should be considered, and allow orchestration of containerized and VM-based VNFs in a similar way. Nobody wants to create a new MANO stack for CNFs.

Reply:
I fully agree that we should remain aligned with ETSI definitions/specifications, so CNF being only a possible implementation of ETSI-defined VNF, the current VNF/CNF-related definitions should be preserved.

Reply:
I'd suggest "remain aligned with ETSI defs/specs where it makes sense", i.e. take a non-dogmatic approach here.

For CNTT to have impact, it needs to be a) as concrete as possible, b) aligned with the reality where the industry has been/is moving, and c) timely. Being dogmatic on following ETSI NFV would be counter-productive to this. For sure we don't want to repeat another IFA005/006 odyssey.

I'm fully aware of ETSI NFV's _intention_ when defining VNFs/VNFCs as agnostic towards VM vs container deployment. After all, I was the one leading the definition back then.
The reality is, though, that many of the following ETSI NFV specs were written with "VNF = VM-deployed" in mind. Do the experiment and mentally substitute VNFs --> containerized app and you'll easily find technical flaws all over the specs.

More importantly, though:
Suggesting containerized NFs should be managed and orchestrated the same as VM-based NFs completely misses the point of Kubernetes cloud-native operations!
Note, for example, that Kubernetes *is* a VNFM (cf. Operator pattern) and actually ticks all the boxes that after 6 years we as an industry failed to deliver on the VM side of the aisle. Let's not lose that!

So I fully agree with the sentiments that "MANO aspects should be considered" and "nobody wants to create a new MANO stack for CNFs." This is exactly why we need to remain aligned with / address potential gaps in the CNCF's solutions.